src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2805,33 +2803,52 @@ private static bool IsIncompleteParseException(Exception e)
                 return remoteException.ErrorRecord.CategoryInfo.Reason == nameof(IncompleteParseException);
             }
 
-            private void EvaluateSuggestions(ConsoleHostUserInterface ui)
+            private void EvaluateFeedbacks(ConsoleHostUserInterface ui)
             {
                 // Output any training suggestions
                 try
                 {
-                    List<string> suggestions = HostUtilities.GetSuggestion(_parent.Runspace);
-
-                    if (suggestions.Count > 0)
+                    List<FeedbackEntry> feedbacks = FeedbackHub.GetFeedback(_parent.Runspace);
+                    if (feedbacks is null || feedbacks.Count == 0)
                     {
-                        ui.WriteLine();
+                        return;
                     }
 
-                    bool first = true;
-                    foreach (string suggestion in suggestions)
+                    // Feedback section starts with a new line.
+                    ui.WriteLine();
+
+                    const string Indentation = "  ";
+                    int count = 0;
+                    var output = new StringBuilder();
+
+                    foreach (FeedbackEntry entry in feedbacks)
                     {
-                        if (!first)
-                            ui.WriteLine();
+                        if (count > 0)
+                        {
+                            output.AppendLine();
+                        }
+
+                        output.Append("Suggestion [")
```

Comment:
Consider adding a new PSStyle formatting for rendering suggestions

Reply:
Styles added to `$PSStyle`.